openpype/pipeline/colorspace.py

```diff
@@ -533,29 +533,78 @@ def get_ocio_config_colorspaces(config_path):
     return CachedData.ocio_config_colorspaces[config_path]
 
 
-def get_labeled_colorspaces(
-    config_path,
+def convert_colorspace_enumerator_item(
+    colorspace_enum_item,
+    config_items
+):
+    """Convert colorspace enumerator item to dictionary
+
+    Args:
+        colorspace_item (str): colorspace and family in couple
+        config_items (dict[str,dict]): colorspace data
+
+    Returns:
+        dict: colorspace data
+    """
+    # split string with `::` separator and set first as key and second as value
+    item_type, item_name = colorspace_enum_item.split("::")
+
+    item_data = None
+    if item_type == "aliases":
+        # loop through all colorspaces and find matching alias
+        for name, _data in config_items.get("colorspaces", {}).items():
+            if item_name in _data.get("aliases", []):
+                item_data = deepcopy(_data)
+                item_data.update({
+                    "name": name,
+                    "type": "colorspace"
+                })
+                break
+    else:
+        # find matching colorspace item found in labeled_colorspaces
+        item_data = config_items.get(item_type, {}).get(item_name)
+        if item_data:
+            item_data = deepcopy(item_data)
+            item_data.update({
+                "name": item_name,
+                "type": item_type
+            })
+
+    # raise exception if item is not found
+    if not item_data:
+        message_config_keys = ", ".join(
+            "'{}':{}".format(key, set(config_items.get(key, {}).keys())) for key in config_items.keys()
+        )
+        raise KeyError(
+            "Missing colorspace item '{}' in config data: [{}]".format(
+                colorspace_enum_item, message_config_keys
+            )
+        )
+
+    return item_data
+
+
+def get_colorspaces_enumerator_items(
+    config_items,
     include_aliases=False,
     include_looks=False,
     include_roles=False,
     include_display_views=False
-
 ):
     """Get all colorspace data with labels
 
     Wrapper function for aggregating all names and its families.
     Families can be used for building menu and submenus in gui.
 
     Args:
-        config_path (str): path leading to config.ocio file
+        config_items (dict[str,dict]): colorspace data
```

Comment:
It's a bit unclear how I generate the `config_items` entry I need here from an OCIO file. Which function does that? Is it `get_ocio_config_colorspaces(filepath)`?